### DISCUSSION TITLE
How about adding a new example usage that reminds the user to utilize the escape method for the connection string?

### DISCUSSION THREAD

**Discussion by karminski** [jackc/pgx#1898] [General]
Hi, I have just added some new lines to the example usage code in the README.md file to remind users to utilize the escape method for the connection string.

For certain Golang database driver libraries, such as [go-sql-driver](https://github.com/go-sql-driver/mysql?tab=readme-ov-file#password) the connection string is automatically escaped by the library itself. Therefore, I believe it is crucial to inform users whether the connection string requires escaping or not. 

And according history issue [https://github.com/jackc/pgx/issues/1285](https://github.com/jackc/pgx/issues/1285), some users may not be aware of the need to escape their passwords.

Pull requests are here: [https://github.com/jackc/pgx/pull/1897](https://github.com/jackc/pgx/pull/1897)

**Comment by jackc** [comment]
Responded on the PR.
